src/System.Security.Permissions/ref/System.Security.Permissions.Forwards.cs

```diff
@@ -18,3 +18,6 @@
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Security.PermissionSet))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Security.Permissions.PermissionState))]
 #endif
+#if netcoreapp
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Xaml.Permissions.XamlAccessLevel))]
```

Comment:
This type-forward is not needed since the type has only lived in System.Security.Permissions for 3 weeks.